docs/Lama2/docs/explanation/l2format.md

```diff
@@ -93,6 +90,27 @@ export PHOTO=`base64 aadhaarlarge.jpg`
 export AHOST="http://localhost:8000"
 ```
 
+Get [Source Files](https://github.com/HexmosTech/Lama2/tree/main/examples/0004_env_switch_root)
+
+### API environment variables can be defined at project level using `l2config.env`
+
+_l2_ loads local variables from `l2.env`.
```

Comment:
This  is confusing to the end user. You mean - at file level or at variable?

The fact is, we load variables from both files. If there is ever a conflict, the way prioritize selection is:

1. VM vars
2. `l2.env` vars
3. `l2config.env` vars